Libplanet/Blockchain/BlockChain.cs

```diff
@@ -1662,6 +1663,20 @@ internal void CleanupBlockCommitStore(long limit)
                         string.Empty, (s, key) => s + key.ValidatorPublicKey + ", \n"));
             }
 
+            // TODO: Add an unit case for this method.
+            BigInteger commitPower = blockCommit.Votes.Aggregate(
+                BigInteger.Zero,
+                (power, vote) => power + (vote.Flag == VoteFlag.PreCommit
+                    ? validators.GetValidator(vote.ValidatorPublicKey).Power
+                    : BigInteger.Zero));
+            if (validators.TotalPower * 2 / 3 > commitPower)
```

Comment:
Doesn't `ValidatorSet` have `TwoThirdsPower` property?

Reply:
Also, I think it should be `>=`.

Reply:
Amended!